recipes/iptkl/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "iptkl" %}
+{% set version = "0.6.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 340dc6265e64a3341f0db05151edf251e1e6c8476401d2a10194fbbaeb16a7db
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - biopython
+    - bokeh
+    - colour
+    - h5py
+    - holoviews
+    - logomaker
+    - lxml
+    - matplotlib
+    - mhcnames
+    - nglview
+    - numba
+    - numba ==0.53.1
```

Comment:
you have two time numba here ... is the strict pin needed?

Reply:
Thanks a lot for pointing this out, not really, it was just a copy-and-paste typo from two different yaml files that I working on parallel, I have update the yaml files according, thank you very much.